great_expectations/execution_engine/execution_engine.py

```diff
@@ -592,7 +592,7 @@ class MetricDomainTypes.
 
     @staticmethod
     def _split_table_metric_domain_kwargs(
-        domain_kwargs: Dict,
+        domain_kwargs: dict,
```

Comment:
I don't mind either way but do we have a convention for `dict` vs `Dict`? Don't think it's worth splitting hairs over (as long as they both mean `Dict[str, Any]`)

Reply:
@cdkini I think that `Dict` without the types for key and value is wrong.